### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Basile Buxtorf and Dorian Kury

### DISCUSSION THREAD

**Discussion by BasileBux** [heig-vd-dai-course/heig-vd-dai-course#355] [Show and tell]
A cli tool to modify audio files. The modifications will be reverse,  volume,  speed. On the whole file or only on certain parts you can chose.

**Comment by ludelafo** [comment]
Looks good to us, thank you!

Please be aware of the time available. Go to the essentials and add more features if the time allows it.

Maybe we would only focus to apply a filter to the entire audio file, as opposed to only a section of the song/music.

**Comment by BasileBux** [comment]
As asked for the submitting: 
- [repo](https://github.com/BasileBux/java-io-audio-files)
- commit hash: 
```
d0dde7ed7b97bbb5367b3f5ed0fa087f3a25cddd
```
